pkg/transforms/vapbinding.go

```diff
@@ -0,0 +1,46 @@
+/*
+IBM Confidential
+OCO Source Materials
+(C) Copyright IBM Corporation 2019 All Rights Reserved
+The source code for this program is not published or otherwise divested of its trade secrets,
+irrespective of what has been deposited with the U.S. Copyright Office.
+Copyright (c) 2020 Red Hat, Inc.
+*/
+// Copyright Contributors to the Open Cluster Management project
+
+package transforms
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+type VapBindingResource struct {
+	node Node
+}
+
+func VapBindingResourceBuilder(v *unstructured.Unstructured) *VapBindingResource {
```

Comment:
I could use `k8s.io/api` v0.30 but I had to change prow, other libraries(a lot) and go version. Therefore, I rather chose to use `unstruntured`